php/ext/google/protobuf/php-upb.h

```diff
@@ -115,7 +115,11 @@ int msvc_vsnprintf(char* s, size_t n, const char* format, va_list arg);
  */
 #define _upb_snprintf __builtin_snprintf
 #define _upb_vsnprintf __builtin_vsnprintf
+#if __GNUC__ >= 3
```

Comment:
I'm not sure if protobuf supports GCC < 3.0, but if it doesn't, we can drop this.